bottles/frontend/ui/details-bottle.blp

```diff
@@ -40,6 +40,17 @@ Popover pop_context {
     Separator {
     }
 
+    .GtkModelButton btn_toggle_removed {
+      text: _("Toggle Hidden");
```

Comment:
It's difficult to tell whether something is toggled or not without a clear indication. Should this toggle be renamed to "Show Hidden", when it's toggled off, and "Hide Hidden", when it's toggled on?

Reply:
"Hide Hidden" is kinda whacky

Reply:
157305ea16772f51af8fb2761ed0d5b88cf6c5c0